db/migrate/20230926163846_create_bulk_discounts.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBulkDiscounts < ActiveRecord::Migration[7.0]
```

Comment:
Hooray! No weird migration issues with merchant getting a merchant_id that breaks all the tests!